### DISCUSSION TITLE
PDF Download possibly broken?

### DISCUSSION THREAD

**Discussion by rajendrapande** [lencx/ChatGPT#910] [General]
Not sure how to and what to look for but the pdf button now does nothing. Only the first (md) and the fourth (Regenerate) options work

**Comment by vuxxs** [comment]
Which PDF button? Are you referring to an extension or a separate website?

**Comment by rajendrapande** [comment]
![image](https://github.com/lencx/ChatGPT/assets/19562017/ccb93536-0362-499d-9ba9-0251c753e657)
I was talking about the four buttons - Markdown, Generate PNG, Download PDF and Refresh Page. 
Though I am happy to report that with version 1.0 its working now. Though the default location is now the "down;oad" folder (different from what it was)
Thanks
